docs/getting-started/my-account/README.md

```diff
@@ -76,6 +76,12 @@ This mode is recommended for users with visuals impairment.
 
 ![High contrast mode in OpenProject](openproject_my_account_high_contrast_mode.png)
 
+### Select the dark mode
+
+In the dropdown menu **Mode** you can pick the color mode. The default setting is the **Light mode**. You can also select the **Dark mode**, which will change the color theme of the OpenProject instance for you.
```

Comment:
I don't know whether this is the right place, but somewhere we should mention that the dark mode ignores parts of the configured design. Top header and sidemenu colors are completely ignored and overriden by us for this mode. Only the accent color and the primary button color are kept, but calculated to brighter variants.

Reply:
Thanks for the comment, I'll add it here. For now it's the only place where Dark mode is mentioned.